tests/e2e/detectors/test_data/naming-convention/0.4.25/naming_convention.sol

```diff
@@ -53,6 +53,7 @@ contract Test {
 
 contract T {
     uint private _myPrivateVar;
+    uint internal _myInternalVar;
     uint _myPublicVar;
```

Comment:
```suggestion
    uint  public _myPublicVar;
```

Reply:
Would you make all of the `_myPublicVar`'s public and run `python tests/e2e/detectors/test_detectors.py --compile` to update the compiled artifacts please? You'll also want to run `pytest tests/e2e/detectors/test_detectors.py --insta update` to update the snapshots as the public variable should be included in the snapshot after updating the solidity code.

Reply:
Thanks! 

I updated the variable to `public` and updated the snapshots accordingly.